src/applications/find-forms/components/SearchResult.jsx

```diff
@@ -186,12 +186,27 @@ const SearchResult = ({
               className="fas fa-chevron-circle-right fa-2x vads-u-margin-right--1"
               role="presentation"
             />
-            <span className="vads-u-text-decoration--underline vads-u-font-weight--bold">
-              Go to online tool
-            </span>
-            <span className="vads-u-visibility--screen-reader">
-              for {id} {title}
-            </span>
+            {language === 'es' ? (
+              <>
+                {' '}
+                <span
+                  lang={language}
+                  className="vads-u-text-decoration--underline vads-u-font-weight--bold"
+                >
+                  Llene el formulario VA {id} en línea.
```

Comment:
nit: I think it would make sense to keep the different versions of this text in a constant in case we add more languages.

Reply:
Yeah, I was hoping someone would say something. 
Maybe eventually it can be a matrix/ object or languages. But for now, just a simple constant or obj?

Reply:
I think a simple object works for now 👍